worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -845,6 +845,9 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
     fileInfos.values().asScala.map(_.values().asScala.map(_.getBytesFlushed).sum).sum
   }
 
+  def getActiveShuffleFileCount(): Long = {
+    fileInfos.values().size()
```

Comment:
`private val fileInfos =
    JavaUtils.newConcurrentHashMap[String, ConcurrentHashMap[String, FileInfo]]()`
As you can see the the key is shuffleKey.
So this should be 
`fileInfos.asScala.values.map(_.size()).sum`